recipes/libtorch/all/conanfile.py

```diff
@@ -0,0 +1,667 @@
+
+    @property
+    def _depends_on_pthreadpool(self):
+        return self._is_mobile_os or self._use_nnpack_family
+
+    @property
+    def _depends_on_flatbuffers(self):
+        return not self._is_mobile_os
+
+    @property
+    def _blas_cmake_option_value(self):
+        return {
+            "eigen": "Eigen",
+            "atlas": "ATLAS",
+            "openblas": "OpenBLAS",
+            "mkl": "MKL",
+            "veclib": "vecLib",
+            "flame": "FLAME",
+            "generic": "Generic"
+        }[str(self.options.blas)]
+
+    @property
+    def _use_nnpack_family(self):
+        return any(self.options.get_safe(f"with_{name}") for name in ["nnpack", "qnnpack", "xnnpack"])
+
+    def requirements(self):
+        self.requires("cpuinfo/cci.20231129")
+        self.requires("eigen/3.4.0")
+        # fmt/11.x is not yet supported as of v2.4.0
+        self.requires("fmt/10.2.1", transitive_headers=True, transitive_libs=True)
+        self.requires("foxi/cci.20210217", libs=False)
+        self.requires("onnx/1.16.1", transitive_headers=True, transitive_libs=True)
+        self.requires("protobuf/3.21.12")
+        self.requires("fp16/cci.20210320")
+        self.requires("cpp-httplib/0.16.0")
+        self.requires("libbacktrace/cci.20210118")
+        if self._depends_on_sleef:
+            self.requires("sleef/3.6")
+        if self._depends_on_flatbuffers:
+            self.requires("flatbuffers/24.3.25", libs=False)
+        if self.options.blas == "openblas":
+            # Also provides LAPACK, currently
+            self.requires("openblas/0.3.27")
+        if self.options.with_openmp:
+            self.requires("openmp/system", transitive_headers=True, transitive_libs=True)
+        if self.options.with_fbgemm:
+            self.requires("fbgemm/0.8.0", transitive_headers=True, transitive_libs=True)
+        if self.options.with_gflags:
+            self.requires("gflags/2.2.2")
+        if self.options.with_glog:
+            self.requires("glog/0.7.1", transitive_headers=True, transitive_libs=True)
+        if self.options.get_safe("with_qnnpack"):
+            self.requires("fxdiv/cci.20200417")
+            self.requires("psimd/cci.20200517")
+        if self.options.with_xnnpack:
+            self.requires("xnnpack/cci.20240229")
+        if self.options.with_itt:
+            self.requires("ittapi/3.24.4")
+        if self._depends_on_pthreadpool:
+            self.requires("pthreadpool/cci.20231129")
+        if self.options.get_safe("with_numa"):
+            self.requires("libnuma/2.0.16")
+        if self.options.with_opencl:
+            self.requires("opencl-headers/2023.12.14")
+            self.requires("opencl-icd-loader/2023.12.14")
+        if self.options.with_vulkan:
+            self.requires("vulkan-headers/1.3.268.0")
+            self.requires("vulkan-loader/1.3.268.0")
+        if self.options.with_mimalloc:
+            self.requires("mimalloc/2.1.7")
+
```

Comment:
Build on MacOS needs pybind
```suggestion
        if is_apple_os(self):
            self.requires("pybind11/2.13.6")
```

Reply:
Are you sure it's required specifically on macOS? I might have missed it on Linux due to having it available on my system.

Reply:
I tried it without but had no success. You can have a look to 
https://github.com/joda01/imagec-recipes/actions/runs/12983296316